fc/person.go

```diff
@@ -137,9 +137,11 @@ type Age struct {
 }
 
 type Identifiers struct {
-	Maids     []Maids  `json:"maids"`
-	PersonIds []string `json:"personIds"`
-	RecordIds []string `json:"recordIds"`
+	Maids      []Maids  `json:"maids"`
+	PersonIds  []string `json:"personIds"`
+	RecordIds  []string `json:"recordIds"`
+	LiNonId    []string `json:"li_nonid"`
```

Comment:
I believe @loganbates-fc  allowed for `li_nonid` AND `li_nonId` to both work for this field, so we can make it upper case I if that's preferred for consistency